native/cocos/renderer/gfx-vulkan/VKDevice.cpp

```diff
@@ -620,6 +620,10 @@ void CCVKDevice::acquire(Swapchain *const *swapchains, uint32_t count) {
     vkPresentBarriers.resize(count, presentBarrier);
     for (uint32_t i = 0U; i < count; ++i) {
         auto *swapchain = static_cast<CCVKSwapchain *>(swapchains[i]);
+        if (!swapchain->gpuSwapchain()) {
+            continue;
+        }
```

Comment:
We might change this guard to `CC_EXPECTS(swapchain->gpuSwapchain());`. I believe it is always valid.

Reply:
yes~ agree with it!
change pulled